<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QComboBox" name="cmbKalem"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Kalem</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btSil">
       <property name="text">
        <string>Sil</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="txtTutar"/>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="btYeni">
       <property name="text">
        <string>Yeni</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QTableWidget" name="lstHarcama">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Tutar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cmbAy"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ay</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Seçilen Kayıt Numarası:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="btKaydet">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Harcama Listesi</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblKayit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAyarlar">
    <property name="title">
     <string>Ayarlar</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionKalem_Ekle"/>
   </widget>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionKaydet"/>
    <addaction name="actionG_ncelle"/>
    <addaction name="actionSil"/>
    <addaction name="separator"/>
    <addaction name="actionTemizle"/>
    <addaction name="separator"/>
    <addaction name="action_k"/>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menuAyarlar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionKalem_Ekle">
   <property name="text">
    <string>Kalem Ekle</string>
   </property>
  </action>
  <action name="actionKaydet">
   <property name="text">
    <string>Kaydet</string>
   </property>
  </action>
  <action name="actionG_ncelle">
   <property name="text">
    <string>Güncelle</string>
   </property>
  </action>
  <action name="actionSil">
   <property name="text">
    <string>Sil</string>
   </property>
  </action>
  <action name="actionTemizle">
   <property name="text">
    <string>Temizle</string>
   </property>
  </action>
  <action name="action_k">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
